docs/basic_concepts.rst

```diff
@@ -0,0 +1,172 @@
+
+In PyOTA, ``trits`` are represented as a sequence of numerical values (List[``int``])
+while trytes have their own class called :py:class:`TryteString`.
+
+IOTA token
+----------
+
+The `IOTA token`_ is a `unit`_ of value that can be transferred over an IOTA
+network through `transfer bundles`_.
+
+The IOTA token was launched on the Mainnet in June 2017. At this point, the
+nodes in the network were hard-coded with a total supply of 2,779,530,283 277,761.
+This large supply allows each of the billions of devices, which are expected to
+be a part of the Internet of Things, to have its own wallet and transact
+micropayments with other devices.
+
+Seed
+----
+
+`Seed`_ in IOTA is your unique password. It is the digital key that unlocks
+your safe that holds your tokens, or proves the ownership of messages.
+
+Seeds in PyOTA are always 81 trytes long and may only contain characters from
+the `tryte alphabet`_.
+
+.. warning::
+    Treat your seed(s) the same as you would the password for any other financial service.
+    Never share your seed with anyone and never use online generators to create a seed.
+    The library can help you to create your own locally and it does not require
+    internet connection: :py:meth:`iota.crypto.Seed.random`.
+
+For PyOTA specific implementation details on seeds, see :py:class:`crypto.Seed`.
+
+Address
+-------
+
+To send or receive any transaction (let them be zero-value or value transacitons)
+in IOTA, you will need to specify an `address`_. An address is like a physical mailbox
+on your entrance door: anyone can drop things in your mailbox (send you messages
+or tokens), but only you can empty it (withdraw tokens).
+
+.. warning::
+    Addresses should not be re-used once they are `spent from`_.
+    **You can receive as many transactions to an address as you wish, but only
+    spend from that address once.**
+
+    That being said, if you received money in your mailbox, you better move to a
+    new flat.
+
+Addresses are generated from your seed through cryptographic functions. There
+are 9\ :sup:`57`\  different addresses that one might generate from a seed,
+which is quite a lot. Given your ``seed``, the ``index`` and a ``security level``
+parameters specify which address will be generated from it. The process is
+deterministic, meaning that same input paramteres always generate the same address.
+
+Addresses are 81 trytes long and may contain extra 9 trytes for checksum.
+The checksum may be used to verify that an address is in fact a valid IOTA address.
+
+For PyOTA specific implementation details on addresses, see :py:class:`Address`.
+
+Transaction
+-----------
+
+From the `official IOTA documentation site`_:
+
+*A transaction is a single transfer instruction that can either withdraw IOTA
+tokens from an address, deposit them into an address, or have zero-value
+(contain data, a message, or a signature). If you want to send anything to an
+IOTA network, you must send it to a node as a transaction.*
```

Comment:
Would a [`block-quote`](http://docutils.sourceforge.net/docs/ref/doctree.html#block-quote) be suitable here?

Reply:
Yes, it looks nicer that way.